tests/custom_tests/test_all_model.py

```diff
@@ -36,6 +37,25 @@ def test_custom_segmentation(combined_all_model_fixture):
     ]
     actual_tokens = [t.text for t in doc]
     assert expected_tokens == actual_tokens
-    assert doc.is_parsed
+    assert doc.has_annotation("DEP")
     assert doc[0].dep_ == "ROOT"
     assert doc[0].tag_ == "NN"
+
+def test_full_pipe_serializable(combined_all_model_fixture):
+    text = "Induction of cytokine expression in leukocytes (CEIL) by binding of thrombin-stimulated platelets. BACKGROUND: Activated platelets tether and activate myeloid leukocytes."
+    doc = [doc for doc in combined_all_model_fixture.pipe([text, text], n_process = 2)][0]
+    # If we got here this means that both model is serializable and there is an abbreviation that would break if it wasn't
+    assert len(doc._.abbreviations) > 0
+    abbrev = doc._.abbreviations[0]
+    assert abbrev["short_text"] == "CEIL"
+    assert abbrev["long_text"] == "cytokine expression in leukocytes"
+    assert doc[abbrev["short_start"] : abbrev["short_end"]].text == abbrev["short_text"]
+    assert doc[abbrev["long_start"] : abbrev["long_end"]].text == abbrev["long_text"]
+
+
+@pytest.mark.xfail
```

Comment:
I realized I actually asked for the wrong thing here, because I want to make sure this test fails, not just allow it to fail (which is what xfail does). What we actually want is `pytest.raises`. So:

```suggestion
def test_full_pipe_not_serializable(combined_all_model_fixture_old_abbrev):
    text = "Induction of cytokine expression in leukocytes (CEIL) by binding of thrombin-stimulated platelets. BACKGROUND: Activated platelets tether and activate myeloid leukocytes."
    # This line requires the pipeline to be serializable (because it uses 2 processes), so the test should fail here
    with pytest.raises(TypeError):
        doc = [doc for doc in combined_all_model_fixture_old_abbrev.pipe([text, text], n_process = 2)][0]
```

Reply:
This won't work - the TypeError will get raised inside of a subprocess so it won't get caught, and won't terminate.

Reply:
oh ok, can you just replace with whatever the actual error is?